test/indefinite.js

```diff
@@ -1,119 +1,119 @@
-require('should');
+import assert from 'assert';
+import indefinite from '../dist/indefinite.cjs';
 
 describe('indefinite', () => {
-  const indefinite = require('../lib/indefinite');
 
   context('a word that starts with a vowel', () => {
     it('should be prefixed with an', () => {
-      indefinite('apple').should.equal('an apple');
+      assert(indefinite('apple') === 'an apple');
```

Comment:
I could live with this if I had too, but I'd like to try to see if we can get should to work.